Sources/Sentry/SentryHub.m

```diff
@@ -668,6 +670,17 @@ - (BOOL)envelopeContainsEventWithErrorOrHigher:(NSArray<SentryEnvelopeItem *> *)
     return NO;
 }
 
+- (void)reportFullDisplayed
+{
+#if SENTRY_HAS_UIKIT
```

Comment:
`m`: I'm unsure why CI didn't run yet for this branch. CodeCov should comment that this is uncovered by tests.